pkg/cmd/server/start/start_node.go

```diff
@@ -479,7 +479,10 @@ func StartNode(nodeConfig configapi.NodeConfig, components *utilflags.ComponentF
 		config.EnsureVolumeDir()
 		config.EnsureDocker(docker.NewHelper())
 		config.EnsureLocalQuota(nodeConfig) // must be performed after EnsureVolumeDir
-		config.RunKubelet()
+		go func() {
+			glog.Fatal(runKubeletInProcess(kubeletArgs))
```

Comment:
do we can about gracefully stopping kubelet on SIGTERM?

Reply:
> do we can about gracefully stopping kubelet on SIGTERM?

It looks like we're running it as a separate process, but this is actually building a complete flagset and then directly calling the cobra command.  It's actually part of our process.